playbook-website/app/views/pages/visual_guidelines_partials/_pb_doc_max_width.html.erb

```diff
@@ -1,20 +1,43 @@
-<%= pb_rails("card/card_body", props: { padding_x: "md", padding_top: "md", padding_bottom: "xs"}) do %>
-  <%= pb_rails("caption", props: { text: "Visual Guide", margin_bottom: "xs"}) %>
-  
-  <%= pb_rails("background", props: { background_color: "gradient", padding: "xs", max_width: "xl", margin_bottom: "xs" }) do %> 
-    <%= pb_rails("title", props: { text: "XL", size: 4, classname: "dark" }) %>
-  <% end %>
-
-  <%= pb_rails("background", props: { background_color: "gradient", padding: "xs", max_width: "lg", margin_bottom: "xs" }) do %> 
-    <%= pb_rails("title", props: { text: "LG", size: 4, classname: "dark" }) %>
-  <% end %>
-
-  <%= pb_rails("background", props: { background_color: "gradient", padding: "xs", max_width: "md", margin_bottom: "xs" }) do %> 
-    <%= pb_rails("title", props: { text: "MD", size: 4, classname: "dark" }) %>
-  <% end %>
-
-  <%= pb_rails("background", props: { background_color: "gradient", padding: "xs", max_width: "sm" }) do %> 
-    <%= pb_rails("title", props: { text: "SM", size: 4, classname: "dark" }) %>
-  <% end %>
-
-<% end %>
\ No newline at end of file
+<div class="p_sm">
```

Comment:
This is very out of date. See the example on the left of what you are deleting. Please reconcile.